src/components/DisplayTaxonName.js

```diff
@@ -1,159 +1,67 @@
-  // I don't feel great about disabling this eslint check, but the this
-  // approach does prevent the crash on Android. We should keep in mind that
-  // there may be capitalization issues down the road ~~~kueda 20230110
-  // eslint-disable-next-line no-misleading-character-class
-  const allCasePattern = new RegExp(
-    `[A-z${lowerCaseChars}${upperCaseChars}]`
-  );
-  const firstLetterMatch = s.match( allCasePattern );
-  let firstLetterIndex = firstLetterMatch ? firstLetterMatch.index : 0;
-  // eslint-disable-next-line no-misleading-character-class
-  const leadingContractionPattern = new RegExp(
-    `^[a-z${
-      lowerCaseChars
-    }][’']([A-z${
-      lowerCaseChars
-    }${upperCaseChars}]+)`
-  );
-  const leadingContractionMatch = s.match( leadingContractionPattern );
-  if ( leadingContractionMatch ) {
-    firstLetterIndex = s.indexOf( leadingContractionMatch[1] );
-  }
-  return (
-    s.slice( 0, firstLetterIndex )
-    + s[firstLetterIndex].toUpperCase()
-    + s.slice( firstLetterIndex + 1 )
-  );
-};
-
-const capitalizeCommonName = name => {
-  if ( !name ) {
-    return name;
+const styles = StyleSheet.create( {
+  commonName: {
+    maxWidth: "100%"
+  },
+  scientificName: {
+    fontStyle: "italic"
```

Comment:
italic fonts need to be added

Reply:
https://github.com/inaturalist/iNaturalistReactNative/issues/449